Executable/README.md

```diff
@@ -0,0 +1,3 @@
+# Usage example
+
+.\hooker.exe "full dll path" "process name"
```

Comment:
It is 32-bit only executable and binaries? Can we add both 32-bit and 64-bit binaries?

Reply:
its old readme, now i'll add python injector, i think we can run injector.py as another process (x32 or x64) depends of target app, and if user haven't x64 python we should warn about it

MSG struct also have different size for x32 and x64 apps, mb memcopy not better way to parse it in socket listener